app/_layout.tsx

```diff
@@ -1,25 +1,15 @@
-import { FlatList, ScrollView, View } from "react-native";
+import { View } from "react-native";
 import { Image } from "expo-image";
 import { Slot } from "expo-router";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-import { cssInterop, remapProps } from "nativewind";
+import { cssInterop } from "nativewind";
 import { StatusBar } from "expo-status-bar";
 import { useSafeAreaEnv } from "react-native-css-interop/dist/runtime/api";
 import "@/global.css";
 
 // component interops for nativewind - just need these once
 cssInterop(Image, { className: "style" });
 
-remapProps(FlatList, {
```

Comment:
This was a bit odd, but basically this mapping made the styles _not_ work on mobile, and removing it makes them work. I'm guessing with newer versions of NativeWind, these are already remapped?